.github/workflows/pull-request.yml

```
@@ -57,10 +57,24 @@
 
       - name: Check for linting errors in modified files
         if: steps.changed-files.outputs.only_changed != 'true'
-        env: 
+        env:
           CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
-        run: npx eslint ${CHANGED_FILES} && python .github/workflows/eslint_disable_check.py
-        
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:7:14: Double quote to prevent globbing and word splitting

(shellcheck)